tools/benchmark_tool/openvino/tools/benchmark/utils/inputs_filling.py

```diff
@@ -269,7 +269,8 @@ def fill_tensors_with_random(layer):
         if shape:
             input_tensors.append(Tensor(rs.uniform(rand_min, rand_max, list(shape)).astype(dtype)))
         else:
-            input_tensors.append(Tensor(rs.uniform(rand_min, rand_max)))
+            scalar = dtype.type(rs.uniform(rand_min, rand_max))
+            input_tensors.append(Tensor(np.ndarray([], type(scalar), np.array(scalar))))
```

Comment:
```suggestion
            scalar = rs.uniform(rand_min, rand_max)
            input_tensors.append(Tensor(np.ndarray([], dtype, np.array(scalar).astype(dtype))))
```
Shouldn't it be like this? `dtype` is known from the start of the scope and array needs to be casted to correct type.

Reply:
Fixed